users/templates/users/edit_profile.html

```diff
@@ -13,9 +13,34 @@
       </div>
       <br>
       <div class="form-group">
-        <button type = 'submit' class = 'btn btn-outline-secondary'> Submit </button>
+        <!-- Trigger the modal with a button -->
+        <button type="button" class="btn btn-info btn-lg" data-toggle="modal" data-target="#myModal">Submit!</button>
+
+        <!-- Modal -->
+        <div class="modal fade" id="myModal" role="dialog">
+          <div class="modal-dialog">
+
+            <!-- Modal content-->
+            <<div class="modal-content">
+              <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title">Confirmation</h4>
+              </div>
```

Comment:
What does this do? Can it be removed?

Reply:
Seems to be the header for the modal including the little x button to close it. I think it would look nicer with it.

Reply:
But it is right aligned followed with random unstyled text. The footer below also includes a properly styled close button.

Reply:
He needs to switch his button tag and the h4 tag. This only right aligns the "x" but the "Confirmation" would stay on the left. For style-wise, it would be better to have two close buttons, one in the header and the one at the footer.